### ISSUE TITLE
Loss is very low, but the validation set accuracy is poor

### ISSUE THREAD

**Issue by geoexploring** [open-mmlab/mmsegmentation#1204]
hello,

The training set and validation set are [ISPRS Potsdam](https://www2.isprs.org/commissions/comm2/wg4/benchmark/2d-sem-label-potsdam/) (semantic segmentation dataset, with only six categories), and the model is [swin_upernet](https://github.com/SwinTransformer/Swin-Transformer-Semantic-Segmentation). The specific configuration file and training log are as follows: 

[20220113_185513.log](https://github.com/open-mmlab/mmsegmentation/files/7867225/20220113_185513.log),
[upernet_swin_tiny_patch4_window7_512x512.txt](https://github.com/open-mmlab/mmsegmentation/files/7867232/upernet_swin_tiny_patch4_window7_512x512.txt)

During the training, the loss has been decreasing, but the accuracy of the validation set is very low. Could you give me some suggestions?

thank you!

**Comment by MengzhangLI** [comment]
Based on your log, I think it is caused by wrong config setting or dataset customization.

After our pr is merged, you can use our provided model to check the correctness of your dataset preparation.

**Comment by MengzhangLI** [comment]
The unmerged pr is here: https://github.com/open-mmlab/mmsegmentation/pull/1097

You can use trained model and its config to check out whether dataset preparation is correct.
